grails-app/migrations/0.9.x/changelog-2024-04-29-1200-alter-table-picklist-item-add-quantity-picked.groovy

```diff
@@ -0,0 +1,56 @@
+databaseChangeLog = {
+
+    changeSet(author: "anadolny", id: "290420241230-0", objectQuotingStrategy: "LEGACY") {
+        preConditions(onError: "HALT", onFail: "MARK_RAN", onSqlOutput: "IGNORE") {
+            not {
+                columnExists(columnName: "quantity_picked", tableName: "picklist_item")
+            }
+        }
+
+        addColumn(tableName: "picklist_item") {
+            column(name: "quantity_picked", type: "INT")
+        }
+    }
+
+    changeSet(author: "anadolny", id: "290420241230-2", objectQuotingStrategy: "LEGACY") {
+        preConditions(onError: "HALT", onFail: "MARK_RAN", onSqlOutput: "IGNORE") {
+            not {
+                columnExists(columnName: "date_picked", tableName: "picklist_item")
+            }
+        }
+
+        addColumn(tableName: "picklist_item") {
+            column(name: "date_picked", type: "DATETIME")
+        }
+    }
+
+    changeSet(author: "anadolny", id: "290420241230-3", objectQuotingStrategy: "LEGACY") {
+        preConditions(onError: "HALT", onFail: "MARK_RAN", onSqlOutput: "IGNORE") {
+            not {
+                columnExists(columnName: "picker_id", tableName: "picklist_item")
+            }
+        }
+
+        addColumn(tableName: "picklist_item") {
+            column(name: "picker_id", type: "CHAR(38)")
+        }
+    }
+
+    changeSet(author: "anadolny", id: "290420241230-4", objectQuotingStrategy: "LEGACY") {
+        preConditions(onError: "HALT", onFail: "MARK_RAN", onSqlOutput: "IGNORE") {
+            not {
+                foreignKeyConstraintExists(foreignKeyName: "fk_picker_person")
+            }
+        }
+
+        addForeignKeyConstraint(
+                baseColumnNames: "picker_id",
+                baseTableName: "picklist_item",
+                constraintName: "fk_picker_person",
```

Comment:
One thing that caught my eye before merging it. Could you change the name of this constraint to match a convention of having names of base table name and referenced table name -> `fk_picklist_item_person`

Reply:
Actually, your version makes more sense, let me think shortly about this one

Reply:
Yeah I was partially right. We usually do `fk + base table name + base column name (without _id)` for these constraint names, so I think this could be the way for this one, but your version `fk + base column name + reference table name` is ok too I guess

Reply:
As discussed internally on slack we are going to use a convention of `fk_baseTable_baseColumn_referenceTable` if the changelogs are not generated.